File: 2058-concatenation-of-array/concatenation-of-array.go
func getConcatenation(nums []int) []int {
    var n int = len(nums)
    var result = make([]int, n*2)
    for i:=0;i<n;i++{
        result[i] = nums[i]
        result[i + n] = nums[i]
    }
    return result
}

File: 2058-concatenation-of-array/concatenation-of-array.ts
function getConcatenation(nums: number[]): number[] {
    const n: number = nums.length;
    var result: number[] = new Array(n * 2);
    for(let i = 0; i < n; i++){
        result[i] = nums[i];
        result[i + n] = nums[i];
    }
    return result;
};